test/Settings.ts

```diff
@@ -761,451 +350,153 @@ describe('Settings', function () {
-            ),
-            this.wait
-          )
-        ).not.to.be.throw;
+        await SettingsScreen.deleteAccountsButton.click();
+        await ConfirmDeleteAccountsScreen.cancelButton.click();
+        expect(SettingsScreen.root).toBeDisplayed();
       });
 
       it('"Delete all" button is disabled', async function () {
-        await clickDeleteAllBtn.call(this);
-
-        expect(
-          await this.driver
-            .findElement(By.css('[data-testid="resetConfirm"]'))
-            .isEnabled()
-        ).to.be.false;
+        await SettingsScreen.deleteAccountsButton.click();
+        expect(ConfirmDeleteAccountsScreen.deleteAllButton).toBeDisabled();
       });
 
       it('Wrong confirmation phrase displays error', async function () {
-        const defaultPhrase = await this.driver
-          .findElement(By.css('[data-testid="defaultPhrase"]'))
-          .getText();
-        await this.driver
-          .findElement(By.css('[data-testid="confirmPhrase"]'))
-          .sendKeys(defaultPhrase.toLowerCase());
-
-        expect(
-          await this.driver
-            .findElement(By.css('[data-testid="confirmPhraseError"]'))
-            .getText()
-        ).matches(/The phrase is entered incorrectly/i);
+        await ConfirmDeleteAccountsScreen.confirmPhraseInput.setValue(
+          'delete all accounts'
+        );
+        expect(ConfirmDeleteAccountsScreen.deleteAllButton).toBeDisabled();
+        expect(ConfirmDeleteAccountsScreen.confirmPhraseError).toHaveText(
+          'The phrase is entered incorrectly'
+        );
       });
 
       it('Correct confirmation phrase enables "Delete all" button', async function () {
-        const defaultPhrase = await this.driver
-          .findElement(By.css('[data-testid="defaultPhrase"]'))
-          .getText();
-        const phraseInput = this.driver.findElement(
-          By.css('[data-testid="confirmPhrase"]')
+        await ConfirmDeleteAccountsScreen.confirmPhraseInput.setValue(
+          'DELETE ALL ACCOUNTS'
         );
-        await phraseInput.clear();
-        await phraseInput.sendKeys(defaultPhrase);
-
-        expect(
-          await this.driver
-            .findElement(By.css('[data-testid="resetConfirm"]'))
-            .isEnabled()
-        ).to.be.true;
+        expect(ConfirmDeleteAccountsScreen.deleteAllButton).toBeEnabled();
       });
 
       it('Clicking "Delete account" removes all accounts from current network', async function () {
-        await this.driver
-          .findElement(By.css('[data-testid="resetConfirm"]'))
-          .click();
-
-        expect(
-          await this.driver.wait(
-            until.elementLocated(
-              By.xpath("//div[contains(@class, 'content@Welcome-module')]")
-            ),
-            this.wait
-          )
-        ).not.to.be.throw;
+        await ConfirmDeleteAccountsScreen.deleteAllButton.click();
+        await browser.pause(1000);
```

Comment:
Maybe `waitForDisplayed ` the login screen?

Reply:
done